docs/source/tutorials/decorator.rst

```diff
@@ -0,0 +1,181 @@
+Decorators
+==========
+
+When developing a user interface, some common tasks are repetitive and 
+important for the user experience: trigger an action over some widgets 
+when a method is called, catch errors within some method execution and avoid 
+users to click over other buttons while the main process is running.
+
+In this tutorial, you will be introduced to the sepal_ui decorators and how they
+will help you tackle that tasks by taking the following topics:
+
+- Description: ¿what is a decorator and what is its basic syntax?
```

Comment:
no need for the revert "?" in english I'm afraid ;-)

Reply:
hahaha yesss